apps/sms/views/shared/js/startup.js

```diff
@@ -47,7 +47,8 @@ var Startup = {
     '/lib/bridge/bridge.js',
     '/services/js/bridge_service_mixin.js',
     '/services/js/activity/activity_shim.js',
-    '/services/js/activity/activity_client.js'
+    '/services/js/activity/activity_client.js',
+    '/services/js/messaging/messaging_client.js'
```

Comment:
I load the client in the conversation view markup, but maybe we can lazyload messageing client since we don't really need it while view startup.